src/test/features/response/routes/checks/ccj-requested-check.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import '../../../../routes/expectations'
+
+import * as claimStoreServiceMock from '../../../../http-mocks/claim-store'
+import { Paths } from 'dashboard/paths'
+
+const cookieName: string = config.get<string>('session.cookieName')
+
+export function checkCountyCourtJudgmentRequestedGuardGuard (app: any, method: string, pagePath: string) {
+  it(`for ${method} should redirect to your dashboard page when defendant has already responded`, async () => {
```

Comment:
That is misleading. The simulated scenario is when CCJ was requested not that defendant already responded. It should be `for ${method} should redirect to your dashboard page when claimant has already requested CCJ`